lib/clowder-common-ruby/rails_config.rb

```diff
@@ -0,0 +1,123 @@
+module ClowderCommonRuby
+  class RailsConfig
+    attr_reader :config
+    
+    # Map for Kafka configuration
+    def self.configure_kafka(config)
```

Comment:
When defining singleton methods, you can simply put them into a `class << self` block:
```ruby
class BlaBla
  class << self
    def to_h
       ...
    end

    # Strong feature of this syntax is the singleton-private methods.
    private

    def configure_kafka(config)
      ...
    end
  end
```